qtfred/src/CheckedDataListModel.h

```diff
@@ -36,6 +36,7 @@ class CheckedDataListModel : public QAbstractListModel
 		RowData& operator=(RowData&& move) = delete;
 
 		inline U& internalData() {return _internalData;}
+		inline const U& internalData() const {return _internalData;}
```

Comment:
When I first wrote the CheckedDataListModel, I intended it only for the lists of usable ships and weapons, in which case this would indeed have been unmodifiable internal data. I made a separate PR for it, which got merged, in the hopes it might be useful in other dialogs. In hindsight, that was a mistake, as I also found it useful, with some modifications, for an improved (over FRED) mission list. 
This version of the model can take ownership of data, iterate over them, let them be modified externally etc. I believe it would be clearer now to rename anything `internalData` to `managedData`.
This specific method is necessary to iterate over the managed data when only a const reference to the model is available.